acceptable-albatrosses/albatrosses_hub/authentication/tests.py

```diff
@@ -0,0 +1,152 @@
+from django.contrib.auth.models import User, AnonymousUser
+from django.test import TestCase, Client, RequestFactory
+from django.urls import reverse, resolve
+from .views import register_page, login_page, logout_page
+
+
+class AuthTestCase(TestCase):
+    """Class to perform unit tests of auth app."""
+
+    def test_register_url(self):
```

Comment:
A small "pro" tip:

When you are writing unittests using Python's unittest module or a framework that closely relates to that (like Django's testing framework), it's good practice to write a solid one-line test method summary as a docstring.

The reason is that the summary line of a test method will be printed in the output of the test results if the test happens to fail. In a large code base, this could help you identify why multiple tests are suddenly failing right there and then from the output instead of having to look up in the code base what each exact test was supposed to be testing.